Sources/CodeEditTextView/Enums/BracePairHighlight.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  BracePairHighlight.swift
+//  CodeEditTextView
+//
+//  Created by Khan Winter on 5/3/23.
+//
+
+/// An enum representing the type of highlight to use for brace pairs.
+public enum BracePairHighlight {
+    /// Highlight both the opening and closing character in a pair with a bounding box.
+    /// The box will use the theme's text color with some opacity. The boxes will stay on screen until the cursor moves
+    /// away from the brace pair.
+    case box
```

Comment:
I might use the term bordered or outlined instead of box.

Reply:
I might also try adding an underlined style as well 
![image](https://user-images.githubusercontent.com/806104/236710868-61c3331c-8828-4230-b15d-d2b7e2f84750.png)